core/chains/evm/logpoller/log_poller.go

```diff
@@ -896,12 +896,19 @@ func (lp *logPoller) blocksFromFinalizedLogs(ctx context.Context, logs []types.L
 // or if there is an error backfilling.
 func (lp *logPoller) backfill(ctx context.Context, start, end int64) error {
 	batchSize := lp.backfillBatchSize
+	stopCtx, cancel := lp.stopCh.NewCtx()
```

Comment:
Do we need a separate variable?
```suggestion
	ctx, cancel := lp.stopCh.Ctx(ctx)
```

Reply:
@dhaidashenko I'm not sure I fully understand the motivation for the original change. But replacing `stopCtx` with `ctx` definitely seems wrong to me. That would mean we're completely ignoring the context that's passed to `backfill` instead of passing it along to `FilterLogs`

Reply:
What's the motivation for this change? Not sure I follow why we need to call either of these. Isn't the original ctx always a child of the context associated with `stopCh` anyway?